syncback/README.md

```diff
@@ -26,37 +26,42 @@ For every syncback resource you want to create, invoke `syncback` with the follo
 
 * **name (str)**: name of the created local resource
 * **k8s_object (str)**: a Kubernetes object identifier (e.g. `deploy/my-deploy`, `job/my-job`, or a pod ID) that Tilt can use to select a pod. As per the behavior of `kubectl exec`, we will act on the first pod of the specified object, using the first container by default
+* **paths (List[str])**: paths *in the remote container* to sync, relative to `src_dir`. May be files or directories. Note that these must not begin with `./`. Pass an empty list to sync all of src_dir. THIS OPTION RISKS WIPING OUT FILES LOCALLY if the files exist at `target_dir` but not in the container. Tilt will protect some files automatically, but you will probably want to use the `ignores` parameter to explicitly protect other files that exist locally but not on the container.
```

Comment:
> Note that these must not begin with `./`

what's the reasoning behind this restriction? would it make sense to add code to check for this?

Reply:
rsync doesn't like that syntax: it won't end up being evaluated relative to src_dir (it doesn't get rejected outright, idk what it does instead... maybe literally looks for a directory called `.`? but it doesn't translate to `$pwd` in rsync).

Yep, good call, i'll put that check in the code tomoro